web/sites/default/config/views.view.admin_group_content.yml

```diff
@@ -262,9 +262,11 @@ display:
           empty_zero: false
           hide_alter_empty: true
           click_sort_column: value
-          type: string
+          type: boolean
           settings:
-            link_to_entity: true
+            format: custom
+            format_custom_true: '<span class="status-published" title="This page is live"><i class="fa fa-circle" aria-hidden="true"></i></span>'
```

Comment:
We should use the terms "published" and "unpublished" for consistency with other places we use published status.

Reply:
Also, using the word "page" for every type of content is a little problematic. Could we change it to "Has published revision" and "Is unpublished"?

Reply:
@joshuami How about, "Has published revision," and, "No published revision?"

I've updated and redeployed.

Reply:
That works for me.